drivers/net/ethernet/microchip/enc28j60.c

```diff
@@ -1293,13 +1296,15 @@ static netdev_tx_t enc28j60_send_packet(struct sk_buff *skb,
 	 */
 	netif_stop_queue(dev);
 
+	spin_lock(&priv->tx_skb_lock);
```

Comment:
This locking can be omitted, if we use the netdev tx log in cleanup in tx_clear() (see comment above)